.github/issue_template.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Issue Template
+about: Use this template to create all project issues.
+title: ''
+labels: ''
+assignees: ''
+---
+
+**Describe the task that needs to be done.**
+*(If this issue is about a bug, please describe the problem and steps to reproduce the issue)*
```

Comment:
Is it worth saying to include screenshots if necessary/applicable? E.g. of exception stack trace or other stuff

Reply:
Yup seems like a good idea, I will add that in!